graphx/src/main/scala/org/apache/spark/graphx/impl/ReplicatedVertexView.scala

```diff
@@ -41,8 +41,8 @@ class ReplicatedVertexView[VD: ClassTag, ED: ClassTag](
    * shipping level.
    */
   def withEdges[VD2: ClassTag, ED2: ClassTag](
-      edges_ : EdgeRDDImpl[ED2, VD2]): ReplicatedVertexView[VD2, ED2] = {
-    new ReplicatedVertexView(edges_, hasSrcId, hasDstId)
+      _edges: EdgeRDDImpl[ED2, VD2]): ReplicatedVertexView[VD2, ED2] = {
```

Comment:
I noticed we need a white space if the name of variables ends with "_".
Those variables might be used internally so can we rename to which like prepended `_` right?